scripts/skills/actives/rf_sling_acid_flask_skill.nut

```diff
@@ -0,0 +1,9 @@
+this.rf_sling_acid_flask_skill <- this.inherit("scripts/skills/actives/throw_acid_flask", {
+	function create()
+	{
+        this.throw_acid_flask.create();
+        this.m.Icon = "skills/sling_acid_flask.png";
+        this.m.IconDisabled = "skills/sling_acid_flask_sw.png";
```

Comment:
We should discuss, with @Enduriel, if we want the `_sw` or the `_bw` for black & white icons. Vanilla uses `_sw` because of "Schwarzweiss` from German. I've been using `_bw` for BlackWhite due to English.

Reply:
>  prefer _bw for icons
> Acc
> no
> I take that back
> go with vanilla for everything
> so
> _sw
- Enduriel

So I will mark this as resolved. It Icons will keep **sw** in their name